test/test_helper.exs

```diff
@@ -1,4 +1,5 @@
-ExUnit.start()
+timeout = 1000 # ms
+ExUnit.start(timeout: timeout)
```

Comment:
✍️ The default timeout is 60s, so I reduced it to 1s so that tests would fail out more quickly.